src/spatio_temporal_voxel_layer.cpp

```diff
@@ -681,6 +683,11 @@ void SpatioTemporalVoxelLayer::UpdateROSCostmap(double* min_x, double* min_y, \
       touch(it->first.x, it->first.y, min_x, min_y, max_x, max_y);
     }
   }
+
+  for (const auto& cell: cleared_cells)
```

Comment:
Just for styling, can we iterate through this using iterators? It makes life easy for trying different container types for run-time speeds

Reply:
Fixed